Bio/SwissProt/__init__.py

```diff
@@ -136,10 +136,18 @@ def read(handle):
         raise ValueError("More than one SwissProt record found")
     return record
 
+def strip_evidence(value):
+    """Strip evidence codes from values.
+    ORFNames=N310_11497 {ECO:0000313|EMBL:KFP74115.1}; --> ORFNames=N310_11497;
+    SubName: Full=Tumor protein 63 {ECO:0000313|EMBL:KFP74115.1}; --> SubName: Full=Tumor protein 63;
+    """
+    # this should work also for lines with multiple entries / evidence codes
+    if ' {' in value:
```

Comment:
Do you think we need to be more defensive and add something like ``and value.endswith('};')`` here?